Makefile

```diff
@@ -0,0 +1,41 @@
+coverage-all:
+		coverage erase
+		coverage run --source template_handler -m unittest
+		coverage xml
+
+coverage: coverage-all
+		coverage report --show-missing
+
+test:
+	    python -m pytest --junitxml=test-reports/junit.xml
+lint:
+	    pre-commit run --all-files --show-diff-on-failure
+
+acceptance-test:
+	    behave acceptance-tests/
```

Comment:
Neither behave nor the acceptance-tests directory exist in this repo.

Reply:
removed